index.d.ts

```diff
@@ -68,11 +72,10 @@ export interface RunCmd<SuccessAction extends Action, FailAction extends Action
 }
 
 //deprecated types
-export type SequenceCmd<A extends Action> = ListCmd<A>;
-export type BatchCmd<A extends Action> = ListCmd<A>;
-
+export type SequenceCmd<A extends Action = never> = ListCmd<A>;
```

Comment:
batch and sequence (the cmds, not the options on Cmd.list) can actually just be deleted. they were removed in v5 but i guess we forgot the types.